.github/workflows/converter_tests.yaml

```diff
@@ -1,34 +1,65 @@
 name: Converter tests
 on: [push, pull_request]
 jobs:
-
   test:
     name: Test
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
     strategy:
       matrix:
         python-version: ['3.7']
     steps:
-    - uses: actions/checkout@v2
-      with:
-        path: 'galaxy root'
-    - name: Clone galaxyproject/galaxy-test-data
-      uses: actions/checkout@v2
-      with:
-        repository: galaxyproject/galaxy-test-data
-        path: galaxy-test-data
-    - uses: actions/setup-python@v1
-      with:
-        python-version: ${{ matrix.python-version }}
-    - name: Cache venv dir
-      uses: actions/cache@v1
-      id: pip-cache
-      with:
-        path: ~/.cache/pip
-        key: pip-cache-${{ matrix.python-version }}-${{ hashFiles('galaxy root/requirements.txt') }}
-    - name: Move test data
-      run: rsync -av --remove-source-files --exclude .git galaxy-test-data/ 'galaxy root/test-data/'
-    - name: Install planemo
-      run: pip install planemo
-    - name: Run tests
-      run: planemo test --galaxy_python_version ${{ matrix.python-version }} --galaxy_root 'galaxy root' 'galaxy root'/lib/galaxy/datatypes/converters/*xml
+      - uses: actions/checkout@v2
+        with:
+          path: 'galaxy root'
+      - name: Clone galaxyproject/galaxy-test-data
+        uses: actions/checkout@v2
+        with:
+          repository: galaxyproject/galaxy-test-data
+          path: galaxy-test-data
+      - uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Cache venv dir
+        uses: actions/cache@v1
+        id: pip-cache
+        with:
+          path: ~/.cache/pip
+          key: pip-cache-${{ matrix.python-version }}-${{ hashFiles('galaxy root/requirements.txt') }}
+      - name: Move test data
+        run: rsync -av --remove-source-files --exclude .git galaxy-test-data/ 'galaxy root/test-data/'
+      - name: Install planemo
+        run: pip install planemo
+      - name: Install jq
+        run: sudo apt-get install jq
+      - name: Determine converters to check
+        run: |
+          ls 'galaxy root'/lib/galaxy/datatypes/converters/*xml | grep -v -f 'galaxy root'/lib/galaxy/datatypes/converters/.tt_skip > tool_list.txt
+          echo "Skipping checks for the following converters:"
+          ls 'galaxy root'/lib/galaxy/datatypes/converters/*xml | grep -f 'galaxy root'/lib/galaxy/datatypes/converters/.tt_skip
+          echo "Checking only the following converters:"
+          cat tool_list.txt
+      - name: Lint converters
+        run: |
+          mapfile -t TOOL_ARRAY < tool_list.txt
+          for CONVERTER in "${TOOL_ARRAY[@]}"; do
```

Comment:
I think it would be good to have an option in Planemo to take the list of tools to lint/test from a file (in addition to passing them directly as parameters on the command-line).
We would still need to merge reports when we run tests in parallel, but it would simplify the CI code nonetheless. Thoughts?